src/foam/u2/DetailView.js

```diff
@@ -70,10 +70,10 @@ foam.CLASS({
       // TODO: Make an FObjectArray when it validates properly
       preSet: function(_, ps) {
         foam.assert(ps, 'Properties required.');
-        for ( var i = 0 ; i < ps.length ; i++ ) {
+        for ( var i = 0; i < ps.length; i++ ) {
           foam.assert(
               foam.core.Property.isInstance(ps[i]),
-              "Non-Property in 'properties' list:",
+              `Non-Property in 'properties' list:`,
```

Comment:
The left line follows our/Google's convention, which is to use single quotes unless your string contains single quotes, in which case you should use double quotes.